src/chrome/content/rules/Ofo.so.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Non-functional hosts
+		Couldn't connect to server:
+			 - cms.ofo.so
+			 - mail.ofo.so
+
+		4xx client error:
+			 - common.ofo.so
+			 - img.ofo.so
+			 - one.ofo.so
```

Comment:
`http://img.ofo.so/apk/`
You can easily search for test urls using search engines.